interfaces/HO-Portal/src/app/components/header/header.component.html

```diff
@@ -1,62 +1,37 @@
-<ion-grid>
-  <ion-row>
-    <ion-col size="1">
-      <ion-button
-        [title]="'header.home'|translate"
-        [attr.aria-label]="'header.home'|translate"
-        routerLink="/home"
-        fill="clear"
-        size="small"
-        color="medium"
-      >
-        <ion-icon name="home" slot="icon-only"></ion-icon>
-      </ion-button>
-    </ion-col>
-    <ion-col>
-      <h1 class="ion-no-margin">
-        {{ title }}
-      </h1>
-      <ion-note>
-        {{ programTitlePortal }}
-        &nbsp;<!-- 'placeholder' to reserve vertical space for async-loaded programTitlePortal -->
-      </ion-note>
-    </ion-col>
-    <ion-col class="ion-text-end">
-      <app-user-state></app-user-state>
-    </ion-col>
-  </ion-row>
-  <ion-row class="ion-align-items-center">
-    <ion-col size="auto">
-      <ion-button
-        [routerLink]="['/program', programId, 'dashboard']"
-        routerLinkActive
-        #rlaDashboard="routerLinkActive"
-        [fill]=" rlaDashboard.isActive ? 'solid' : 'outline'"
-        size="small"
-      >
-        {{ 'header.dashboard'|translate }}
-      </ion-button>
-      <ion-button
-        *ngIf="showManageAidworkers"
-        [disabled]="!canManageAidWorkers()"
-        [routerLink]="['/program', programId, 'aid-workers']"
-        routerLinkActive
-        #rlaAidWorkers="routerLinkActive"
-        [fill]=" rlaAidWorkers.isActive ? 'solid' : 'outline'"
-        size="small"
-      >
-        {{ 'header.aid-workers'|translate }}
-      </ion-button>
-    </ion-col>
-    <ion-col size="auto">
-      {{ 'header.phases'|translate }}:
-    </ion-col>
-    <ion-col class="horizontal-scroll-container">
-      <div class="ion-text-nowrap">
-        <app-phase-navigation
-          [programId]="programId"
-        ></app-phase-navigation>
-      </div>
-    </ion-col>
-  </ion-row>
-</ion-grid>
+<ion-header class="top-header">
+  <ion-grid>
+    <ion-row>
+      <ion-col size="1">
+        <ion-button
+          [title]="startButtonRouterLink === '/home' ? ('header.home'|translate) : ('header.help'|translate)"
+          [attr.aria-label]="startButtonRouterLink === '/home' ? ('header.home'|translate) : ('header.help'|translate)"
+          [routerLink]="startButtonRouterLink"
+          fill="clear"
+          size="small"
+          color="medium"
+        >
+          <ion-icon [name]="startButtonIcon" slot="icon-only"></ion-icon>
```

Comment:
We should not use conditional values for these properties, but use basically 2 buttons and show only 1 based on some condition.

So 2 chunks of HTML to show the "Home"-button and the "Help"-button.
And then some logic elsewhere to set `showHome` and/or `showHelp`.